mypy.ini

```diff
@@ -356,3 +356,30 @@ ignore_missing_imports = True
 
 [mypy-librosa.*]
 ignore_missing_imports = True
+
+[mypy-torch.optim._multi_tensor]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.adam]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.adamw]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.sgd]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.rmsprop]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.rprop]
+ignore_errors = True
+
```

Comment:
@zou3519, sadly thats not a simple fix. Updating [this line](https://github.com/pytorch/pytorch/blob/275a89a7ee45772b5d353ec488ec2cd0c78a69c3/tools/pyi/gen_pyi.py#L135) will trigger a lot of other errors. There are a lot of special casing around mypy and especially for the methods that return list of tensors. 

I will follow up on this if thats ok.

Reply:
Yup, follow up is fine